var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -15,17 +15,6 @@ class PyBglibpy(PythonPackage):
     version('develop', branch='master')
 
     version('4.4.36', commit='07fe9999a137c3741fd95713149a76a202cb7d7a')
```

Comment:
@wvangeit This PR removes the old versions of `py-bluepy` from spack and simplify `package.py`.
Do you agree to remove from the configuration the old versions of `py-bglibpy` that depend on the old version of `py-bluepy`?
Spack builds only the most recent version of each package, so the old versions should not be needed unless other packages depend on them.

Reply:
@anilbey might be able to help out here, right @wvangeit?

Reply:
Let me check if some other spack packages (maybe psp-validation) depend on older versions of bglibpy.

Reply:
They don't depend on a specific version. 
`depends_on('py-bglibpy@4.4.27:4.999', type='run')`

It will be easier to debug the issues on the circuits with only a single version of bglibpy. 
Shall we set an upper bound on the py-bluepy's version btw.?

Reply:
The bluepy dependency in bglibpy in this PR is the same as in
https://bbpcode.epfl.ch/browse/code/sim/BGLibPy/tree/setup.py?h=refs/heads/master
but I think we can put an upper bound on py-bluepy 2.999 (I don't think that bluepy 3 will be released soon, though).

Reply:
Looks good to me.